services/metadactyl-clj/src/metadactyl/metadata/tool_requests.clj

```diff
@@ -78,8 +76,8 @@
   [uuid]
   (let [req (queries/get-tool-request-details uuid)]
     (when (nil? req)
-      (throw+ {:error_code error-codes/ERR_NOT_FOUND
-               :id   (string/upper-case (.toString uuid))}))
+      (throw+ {:type :clojure-commons.exception/not-found
+               :error (str "Could not locate tool with the following id: " (string/upper-case (.toString uuid)))}))
```

Comment:
`str` normally converts its arguments to strings, so you can generally replace `(.toString obj)` with `(str obj)`. After some experimentation it looks like simply `(string/upper-case uuid)` works too.